settings.js

```diff
@@ -140,12 +139,11 @@ window.spinnakerSettings = {
     netflixMode: netflixMode,
     chaosMonkey: chaosEnabled,
     // whether stages affecting infrastructure (like "Create Load Balancer") should be enabled or not
-    infrastructureStages: infrastructureEnabled,
+    infrastructureStages: process.env.INFRA_STAGES === 'enabled',
```

Comment:
This feels like an unrelated change...

Reply:
settings.js isn't included in this change, I'll revert that file.